### PR TITLE
custom preloading serializer code experiment

### PR DESCRIPTION
**Pull request by camallen**
Added a custom panoptes wrapper over restpack serialzier to allow preloading on the page method.
This allows any valid include params to preload their associations and it provides a DSL to wire up preloads for each serializer. 

I've created a code experiment to test this out on a custom route to see the performance of doing this. 

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -208,3 +208,8 @@ Style/MultilineMethodCallIndentation:
   Enabled: false
 Style/FrozenStringLiteralComment:
   Enabled: false
+Style/StringLiterals:
```
Comment (camallen) [inline .rubocop.yml]: happy to extract this to it's own PR
